library/eclipse-mosquitto

```diff
@@ -1,10 +1,10 @@
 Maintainers: Roger Light <roger@atchoo.org> (@ralight)
 GitRepo: https://github.com/eclipse/mosquitto.git
-GitCommit: 095bbb5e441e45538e924d90483dacb51d62b927
+GitCommit: 2e94e5999bc794afe618051176926ed67812106a
```

Comment:
This commit isn't on `master`, so you'll need a `GitFetch: refs/heads/fixes`.

Reply:
Sorry about that. I'll fix it tomorrow morning.